### ISSUE TITLE
Inference for full size image

### ISSUE THREAD

**Issue by AKT-W** [weichaozeng/TextCtrl#9]
Hi Author, 

Could I edit the full size image in inference process.
I read the paper said your works can input full size image and will cropped it and start editing.

Thanks~

**Comment by weichaozeng** [comment]
The method operates specifically on cropped text images, and the current implementation of inference does not include functionality for cropping full-size images. You may first acquire text regions through either manual cropping or a pre-trained text detection model.

**Comment by AKT-W** [comment]
Hi there,

got it, but could you advised what setting you're using in the paper (cropped+paste tools)? thanks a lot

**Comment by weichaozeng** [comment]
Actually it is simply crop and paste. Text images of the test set are cropped according to its positional annotations from source dataset. After editing, the result images are pasted into the full-size images through pixel-wise replacement. Perspective transformation and its reverse are leveraged if the annotations indicate irregular quadrilateral regions. In pratice detection models such as [mmocr](https://github.com/open-mmlab/mmocr) could help to obtain the text position.
